content/management/configuration-api/v3.4/index.mdx

```diff
@@ -329,14 +329,14 @@ Updates the properties of an Agent specified by `id`.
 #### Request
 
 | Parameter                              | Data type  | Required | Notes                                                                              |
-| -------------------------------------- | ---------- | -------- | ---------------------------------------------------------------------------------- |
+|----------------------------------------|------------|----------|------------------------------------------------------------------------------------|
 | `id`                                   | `string`   | Yes      | Agent ID                                                                           |
 | `name`                                 | `string`   | No       | Agent name                                                                         |
 | `role`                                 | `string`   | No       | Agent role, should be one of the following: `viceowner`, `administrator`, `normal` |
 | `avatar_path`                          | `string`   | No       | URL path for Agent avatar                                                          |
 | `job_title`                            | `string`   | No       | Agent's job title                                                                  |
 | `mobile`                               | `string`   | No       | Agent's mobile number                                                              |
-| `max_chats_count`                      | `int`      | No       | Agent's maximum number of concurrent chats                                         |
+| `max_chats_count`                      | `int`      | No       | Agent's maximum number of concurrent. Limited to 100. chats                        |
```

Comment:
Why no info about the default value here? 🤔

Reply:
It's on purpose. This is an update, so the default is no change here.